web/requirements.txt

```diff
@@ -1,2 +1,3 @@
 Flask==2.3.2
 gunicorn==20.1.0
+diagrams
```

Comment:
Yes and No... if you noticed I use a fork I called diagrams-xtd (that is in source code or should be...) there's a bunch of changes I made to make it work with flask, using the original diagrams lib will broke a lot.